MD-Verlet-Integrator/md_helper.py

```diff
@@ -0,0 +1,64 @@
+# Helper "Library" to compute Molecular Dynamics (MD) Trajectories
+# Numpy and OS python modules are required
+#
+# MD and Velocity Verlet Algorithms were taken from: :
+# Computational Soft Matter:  From Synthetic Polymers to Proteins, Lecture Notes,
+# Norbert Attig, Kurt Binder, Helmut Grubmuller, Kurt Kremer (Eds.),
+# John von Neumann Institute for Computing, Julich,
+# NIC Series, Vol. 23, ISBN 3-00-012641-4, pp. 1-28, 2004
+#
+# Created by: Leonardo dos Anjos Cunha
+# Date: 05/16/17
+# License: GPL v3.0
+
+import psi4
+import numpy as np
+import os
+
+# Function: md_trajectories
```

Comment:
For this and `integrator()`, the function documentation should be contained in docstrings like
```
def integrator(int_alg, timestep, pos, veloc, accel, molec, grad_method)
    """Integrator engine for MD timestep propogation.
    ...
    """
```
More info on docstring conventions can be found [here](https://www.python.org/dev/peps/pep-0257/).